### DISCUSSION TITLE
Site title

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5907] [General]
Hi,
How can I change only the title of page?
I want to have a certain name for page title and a different name at the login screen.
Thanks.

**Comment by si458** [comment]
https://ylianst.github.io/MeshCentral/meshcentral/customization/#web-branding

**Reply by si458** [reply]
you can change the title and title2 but sadly the title applies to ALL pages
for example `MyComputer1 - Mygroup1 - MyTitle` when on a device page OR `MyTitle` when on the computer lists

**Reply by ghost** [reply]
I didn't understand how to only change the page title as Title changes the page title and is also the main title in the login screen

**Reply by si458** [reply]
You can't... the title value applies to the home page and the main webpages shown
